src/renderers/dom/client/ReactMount.js

```diff
@@ -432,14 +431,9 @@ var ReactMount = {
       'for your app.'
     );
 
-    var nextWrappedElement = ReactElement(
+    var nextWrappedElement = ReactElement.createElement(
       TopLevelWrapper,
-      null,
-      null,
-      null,
-      null,
-      null,
-      nextElement
+      { child: nextElement }
```

Comment:
This will break the devtools. Sorry.

https://github.com/facebook/react-devtools/blob/96fcb41fea91c3a03219851512e3e305af9fa7f0/backend/getData.js#L74-L78